### PR TITLE
test: skip dedupe modules stderr matching on Node 10

### PR DESCRIPTION
**Pull request by alan-agius4**
This is really flakey on CI Node.js 10.

### CODE REVIEW

File: tests/legacy-cli/e2e/tests/misc/dedupe-duplicate-modules.ts [hunk 1]
```diff
@@ -4,38 +4,39 @@ import { updateJsonFile } from '../../utils/project';
...
+      })
     `);
 
-    const { stderr } = await ng('build', '--verbose', '--no-vendor-chunk', '--no-progress');
-    if (!/\[DedupeModuleResolvePlugin\]:.+tslib-1-copy -> .+tslib-1/.test(stderr)) {
-        throw new Error('Expected stderr to contain [DedupeModuleResolvePlugin] log for tslib.');
-    }
+  const { stderr } = await ng('build', '--verbose', '--no-vendor-chunk', '--no-progress');
+  const [major] = process.versions.node.split('.').map(Number);
+  if (major > 10 && !/\[DedupeModuleResolvePlugin\]:.+tslib-1-copy -> .+tslib-1/.test(stderr)) {
```
Comment (kyliau) [inline tests/legacy-cli/e2e/tests/misc/dedupe-duplicate-modules.ts:34]: So effectively we're skipping this test in Node 10?
Do you know why it's failing specifically in Node 10?

Comment (alan-agius4) [reply tests/legacy-cli/e2e/tests/misc/dedupe-duplicate-modules.ts:34]: We skip the scrapping of stderr, I didn’t manage to replicate the flake locally on Node 10.

Also, actually a flake not a constant failure

Comment (alan-agius4) [reply tests/legacy-cli/e2e/tests/misc/dedupe-duplicate-modules.ts:34]: Actually with this change I am seeing the failure on Node 12 as well!

Comment (kyliau) [reply tests/legacy-cli/e2e/tests/misc/dedupe-duplicate-modules.ts:34]: OK, so it looks like we'll have to do more investigation and figure out the root cause.

**Pull request closed without merging**